Makefile

```diff
@@ -8,7 +8,7 @@ PYTEST_EXTRA_ARGS := --flake8 --isort --doctest-modules --cov-report term-missin
 
 SHELL := bash
 PYTHON := python
-PIP := pip
+PIP := pip3
```

Comment:
Quick question: Changing this line to pip3 isnt a matter of default config? If we're using venv, should't we have `pip` as the default?

Reply:
@mbeilin I think changing `pip` to `pip3` will not solve the actual issue because if we are using `virtualenv` then it may be different, so updating the documentation is much better than changing the `pip`.

Reply:
changed it back to pip